integration-tests/test_ui.py

```diff
@@ -42,7 +42,12 @@ async def local_hub_local_binder(request):
                 break
         except requests.exceptions.ConnectionError:
             pass
-        await asyncio.sleep(1)
+        # We use a increased linear time between requests
+        # because BinderHub server might require more than 10s to start.
+        time.sleep(i)
```

Comment:
We should move the `sleep` call to the start of the loop before the `try`, otherwise we end up with a delay at the end without a check.